views/djornl/djornl_gene_view.json

```diff
@@ -0,0 +1,82 @@
+{
+  "writebufferIdle": 64,
+  "writebufferActive": 0,
+  "type": "arangosearch",
+  "primarySort": [],
+  "writebufferSizeMax": 33554432,
+  "commitIntervalMsec": 1000,
+  "consolidationPolicy": {
+    "type": "bytes_accum",
+    "threshold": 0.1
+  },
+  "globallyUniqueId": "c98232107/",
+  "cleanupIntervalStep": 2,
+  "id": "98232107",
+  "links": {
+    "djornl_gene": {
+      "analyzers": [
+        "identity"
+      ],
+      "fields": {
+        "transcript": {},
```

Comment:
I'd add this to the search fields, and the same with `go_terms`.

Reply:
By "this" do you mean the transcript field? If the value is an empty object, the default analyzer is "identity" which will still index the value but will not apply any text processing. I can add go_terms with an "identity" index, and i'll add both to the query

Reply:
Oh it looks like go_terms is already indexed. I just pushed a commit so that they are both in the query. Tested manually and they work.